code/drasil-example/Drasil/DblPendulum/Body.hs

```diff
@@ -88,10 +88,10 @@ mkSRS = [RefSec $      --This creates the Reference section of the SRS
 
 justification :: Sentence
 justification = foldlSent [S "A", phrase pendulum, S "consists" `sOf` S "mass", 
-                            S "attached to the end of a rod" `sC` S "its moving curve" `sIs`
-                            S "highly sensitive to initial conditions" +:+ S "Therefore" `sC`
+                            S "attached to the end of a rod" `andIts` S "moving curve" `sIs`
```

Comment:
improving grammar is a good thing. But I would actually prefer 2 PRs, one that focuses just on fixing the issue, and then another that improves such things. If some of the changes are controversial, then the good changes will still go in quickly, and only a few things can drag on for a while longer. Faster overall progress. 
Don't change anything for this PR, just for future ones.

Reply:
Ah okay, sounds good.